plugins/foreman_discovery/4.0/index.md

```diff
@@ -1,7 +1,9 @@
 ---
 layout: plugin
+pluginname: foreman_discovery
 title: Foreman Discovery 4.0 Manual
 version: 4.0
+warning: old
```

Comment:
Are you sure you want to set this on a version that's in the current stable Foreman repos (1.9)?